TelemetrySite/server/data_upload_scripts/data_upload.py

```diff
@@ -61,20 +62,17 @@ def submit_data(mf4_file, dbc_file, context_data):
 # entries long
 #
 # @param collection_access_event reference to the mongo db connection
-# @param id of the main document to which context was uploaded
 # @param data_values_json data to add to the db
-def upload_data_in_chunks(collection_access_event, object_id, data_values_json):
+def upload_data_in_chunks(new_run_data, data_values_json):
     # divide the data into chunks that are less the 16 mb
     # 150_000 ~< 15 mb but always < 16 mb
     sliced_data = list(sliced(data_values_json, 150_000))
     
     collection_access_messages = create_db_connection()["messages"]
-    
     for data_index in range(0, len(sliced_data)):
-        
+        new_run_data["uploadSection"] = data_index
```

Comment:
Update where the uploadSection is to live inside the event. This should remove the need to upload data in side of a nested event label
```suggestion
        new_run_data["event"]["uploadSection"] = data_index
```